src/plugins/layout/nodeinfo.ts

```diff
@@ -4,6 +4,7 @@ import { shouldCaptureCssRules } from "./states/style";
 import { isCssText } from "./states/text";
 
 export const UnmaskAttribute = "data-clarity-unmask";
```

Comment:
>UnmaskAttribute [](start = 13, length = 15)

These should go into the publicly exported declaration for TypeScript. 
Also, we should have a read me or other documentation for the configuration and custom decoration attributes.